src/Common/Standards/OgcApi.Net.Styles/Controllers/MetadataController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OgcApi.Net.Styles.Model.Metadata;
+
+namespace OgcApi.Net.Styles.Controllers;
+
+[EnableCors("OgcApi")]
+[ApiController]
+[Route("api/ogc/collections")]
```

Comment:
```suggestion
[Route("api/ogc/collections/{collectionId}/styles/{styleId}/metadata")]
```

It would be better to move the routing logic from all methods into the controller parameter